DisCatSharp.Docs/articles/modules/audio/lavalink_v4/advanced.md

```diff
@@ -19,12 +19,12 @@ General config:
 ```yml
 lavalink:
   plugins:
-    - dependency: "com.github.topi314.lavasrc:lavasrc-plugin:4.0.0-beta.3"
+    - dependency: "com.github.topi314.lavasrc:lavasrc-plugin:4.0.1"
       repository: "https://maven.topi.wtf/releases"
 [..]
 plugins:
   lavasrc:
-    providers: # Custom providers for track loading. This is the default
```

Comment:
that should stay btw. it's not meant to be copied but as reference

Reply:
Oh, yeah sorry I accidentally took it out while editing the file 😅